xl/plugins.py

```diff
@@ -55,6 +60,12 @@ def __init__(self, exaile, load=True):
             pass
 
         self.plugindirs = [x for x in self.plugindirs if os.path.exists(x)]
+
+        self.user_installed_plugindir = self.plugindirs[0]
```

Comment:
I don't think using list indices is a good idea here. Since `get_data_dirs` is defined in xdg.py, it seems like it would be better to have a way to return the user data directory and the system data directories instead?

Reply:
I added a function